### DISCUSSION TITLE
How to share bucket between accounts?

### DISCUSSION THREAD

**Discussion by synxroform** [nats-io/nats-server#4612] [Q&A] [answered]
Is it possible to share buckets between accounts similar to streams and topics?

**Comment by jnmoyne** [comment, marked as answer]
Yes. The JetStream KV store functionality is built upon the Stream functionality (you can see the bucket's streams for example using `nats stream ls -a`. So everything you can do with a stream you could do with a bucket. Also, the new subject transformation functionality in streams introduced in 2.10 allows you to source KV buckets (even within the same JS domain where the sourcing and sourced buckets have different names).
